.github/pull_request_template.md

```diff
@@ -1,32 +1,48 @@
-<!-- THE FOLLOWING IS FOR THE PR AUTHOR TO FILL OUT
-PLEASE DO NOT MODIFY THE TEMPLATE BEYOND FILLING OUT THE PROPER SECTIONS -->
-## PR Author Checklist:
-<!--  Please complete all items in list. -->
-- [ ] I have linked PR's from all sub-components involved in section below. <!-- PLEASE DO NOT LINK SUBCOMPONENT ISSUES -->
-- [ ] I am confirming reviews are completed in ALL sub-component PR's.
-- [ ] I have run the full RT suite on either Hera/Hercules AND have attached the log to this PR below this line:
-  - LOG: 
-- [ ] I have added the list of all failed regression tests to "Anticipated changes" section.
-- [ ] I have filled out all sections of the template.
-
-## Description
+<!-- INSTRUCTIONS: 
+- Please fill out all sections of this PR and complete the checklist below
+- Please be as descriptive as possible, this is really important.
+- Please "fill in" checkboxes. Use [X] for a filled in checkbox or leave it [ ] for an empty checkbox
+- Please use github markup as much as possible in linking
+i.e.:
+* Linking to UFSWM PR's and issues add "- #<pr/issue number>"
+* Linking to a subcomponent PR and issues add "- <Group>/<Fork>/pull/#" or "-<Group>/<Fork>/issues/#"
+-->
+## Commit Queue Requirements:
+- [ ] Fill out all sections of this template.
+- [ ] All sub component pull requests have been reviewed by their code managers.
+- [ ] Run the full RT suite (compared to current baselines) on either Hera/Derecho/Hercules AND have committed the log to my PR branch.
+- [ ] Add list of all failed regression tests in "Regression Tests" section.
+
+## PR Information
+
+### Description
 <!-- Provide a detailed description of what this PR does in the space provided below-->
 
+### Commit Message
+<!-- Please provide concise information about the changes in this PR to be used as a commit message for the commit history -->
 
-## Linked Issues and Pull Requests
-### Associated UFSWM Issue to close
-<!-- Example: "- Closes #1698" -->
+### Priority
+- [ ] HIGH
+- [ ] Medium
+- [ ] low
```

Comment:
I think we should really just list a priority level of High and give an example of what would be a High priority. So maybe

### Priority
[ ] Bugfix. This PR contains a bug fix and should be prioritized. 
[ ] High. This PR contains a feature or fix needed for a time-sensitive project (eg, retrospectives, implementations)
[ ] Normal.

Reply:
This does make more sense. We should be holding all PR's to the same priority unless it fits into the top two categories.

Reply:
Do we want to add an item for reasons with "High" priority?

Reply:
I think there should be documented reasoning, yes.